lib/pairing-service.js

```diff
@@ -20,7 +20,7 @@ pairingService.run = () => {
         return;
     }
     pairingService.pollPairingStatus()
-        .then(() => {
+        .then((response) => {
```

Comment:
if the response is not used there is no need to add it, correct?

Reply:
Right now it's not getting used but going forward, we need add support to use Node SDK as a library, in that case, the SDK Wrapper caller is needed to make use of response from pairingService.pollPairingStatus() to true / false for device is paired or not with in given attempts.